doc/source/ray-core/package-ref.rst

```diff
@@ -287,6 +291,16 @@ The Ray Command Line API
    :prog: ray get_head_ip
    :show-nested:
 
+.. _ray-monitor-doc:
+
+.. click:: ray.scripts.scripts:monitor
+   :prog: ray monitor
+   :show-nested:
+
+Session
+~~~~~~~
+This section contains commands for inspecting, debugging, and configuring the current cluster.
```

Comment:
Not sure if this is the right explanation & right set of grouping. Please recommend me if you have a better alternative!

Reply:
what about "Running Clusters" or "Job"? session is not a defined concept

Reply:
Seems like these are about "Debugging applications"?

Reply:
Usage stats are technically not in there. Maybe I can make a new section?

Reply:
I used `Debugging applications` and made a new section for usage stats